package.json

```diff
@@ -24,7 +26,10 @@
   },
   "homepage": "https://github.com/brianc/node-pg-types",
   "devDependencies": {
+    "codecov": "^3.5.0",
     "if-node-version": "^1.1.1",
+    "nyc": "^14.1.1",
+    "open-cli": "^5.0.0",
```

Comment:
:/ https://github.com/sindresorhus/open-cli/blob/46ad34a9a4a19bab1cdba4ee71221e29b9d4aff1/cli.js#L3-L7

Reply:
Yes I wish it didn't include things like that and only supported opening existing files. I put it for convenience as you usually want to view the HTML report after running coverage but could leave it out and have it be done manually.